ansible/roles/operator-pipeline/tasks/operator-hosted-pipeline-trigger.yml

```diff
@@ -0,0 +1,189 @@
+---
+- name: Create trigger enabling running Hosted pipeline via GitHub webhook
+  block:
+    - name: Create Hosted pipeline event listener
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: EventListener
+          metadata:
+            name: operator-hosted-pipeline-github-listener
+            labels:
+              app: operator-pipeline
+              suffix: "{{ suffix }}"
+              env: "{{ env }}"
+          spec:
+            serviceAccountName: pipeline
+            triggers:
+              - name: github-listener
+                interceptors:
+                  - github:
+                      secretRef:
+                        secretName: github-webhook-secret
+                        secretKey: "{{ webhook_secret_key }}"
+                      eventTypes:
+                        - pull_request
+                  - cel:
+                      filter: >-
+                        (
+                          header.match("X-GitHub-Event", "pull_request")
+                          && body.action in ["opened", "reopened", "synchronize"]
+                          && body.pull_request.base.ref == "{{ branch }}"
+                        )
+                bindings:
+                  - ref: operator-hosted-pipeline-trigger-binding
+                template:
+                  name: operator-hosted-pipeline-trigger-template
+
+    - name: Create Hosted pipeline Trigger Binding
+      k8s:
+        state: present
+        namespace: "{{ oc_namespace }}"
+        definition:
+          # To see the PR webhook payload, set the logLevel to debug in configmap config-logging-triggers
+          # Other GitHub PR webhook payload example: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
+          apiVersion: triggers.tekton.dev/v1alpha1
+          kind: TriggerBinding
+          metadata:
+            name: operator-hosted-pipeline-trigger-binding
+            labels:
+              app: operator-pipeline
+              suffix: "{{ suffix }}"
+              env: "{{ env }}"
+          spec:
+            params:
+              - name: git_pr_branch
+                value: $(body.pull_request.head.ref)
+              - name: git_pr_title
+                value: $(body.pull_request.title)
+              - name: git_pr_url
+                value: $(body.pull_request.html_url)
+              - name: git_fork_url
+                value: $(body.pull_request.head.repo.ssh_url)
+              - name: git_repo_url
+                value: $(body.pull_request.base.repo.ssh_url)
+              - name: git_username
+                value: $(body.pull_request.head.user.login)
+              - name: pr_head_label
+                value: $(body.pull_request.head.label)
+              - name: pyxis_url
```

Comment:
Honza change the pipeline yesterday a bit and now it accepts `env` parameter instead of `pyxis_url` and a pipeline itself uses the correct Pyxis URL. Please use the new param instead.